config/environments/demo.rb

```diff
@@ -71,6 +71,7 @@
   # Set to :debug to see everything in the log.
   config.log_level = :info
 
-  # Use a different logger for distributed setups.
-  config.logger = ActiveSupport::TaggedLogging.new(Logger.new(STDOUT))
+  if ENV["SEMANTIC_LOGGER_ENABLED"].blank?
```

Comment:
Is this blank treatment as false a consistent pattern in TT? I often prefer to use something a little richer in case somebody drops a `0` or `false` into the env var or something: https://stackoverflow.com/a/44322375

But I wouldn't boil the ocean over it if this is common already in TT env-var handling.

Reply:
i agree, i was only following the convention that was in place. i am happy to make it a smidge more explicit.

i'll give other folks a chance to chime in.

/cc @samandmoore @smudge @aburgel